### PR TITLE
Modified OMPL default planner handling.

### PR DESCRIPTION
**Pull request by rbbg**
As discussed in #197

### CODE REVIEW

File: moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp [hunk 1]
```diff
@@ -355,6 +355,8 @@ ompl_interface::ModelBasedPlanningContextPtr ompl_interface::PlanningContextMana
     if (pc == planner_configs_.end())
       logWarn("Cannot find planning configuration for group '%s' using planner '%s'. Will use defaults instead.",
               req.group_name.c_str(), req.planner_id.c_str());
+  } else { // No planner_id specified, selecting default
```
Comment (davetcoleman) [inline moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp]: See Travis error for proper format

**Comment by rbbg** [comment]
Snap, I thought I ran format on both files but turns out I only did the one. Anyhow, should work now.

**Comment by davetcoleman** [comment]
I re-added the ``projection_evaluator`` and ran the new [move_group_interface](https://github.com/ros-planning/moveit_tutorials/pull/45) tutorial, and it used KPIECE by default. I then added this patch and it switched to RRTConnect. Great!

**Comment by davetcoleman** [comment]
I do not think this should be backported because it changes core behavior.

**Comment by v4hn** [comment]
Awesome! Thanks!

I agree this should not be backported to keep the current behavior (although I suppose it would actually improve the current state for most setups on indigo too...)

**Pull request merged**